ip-docker/pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
+name = "MM-ip-service"
+version = "0.1.0"
+description = "IP service for MM team in E4E"
+authors = []
+
+[tool.poetry.dependencies]
+python = "^3.9"
+Flask = "2.3.1"
+requests = "2.28.2"
+boto3 = "1.26.120"
+numpy = "1.24.3"
+rasterio = "1.3.6"
+rioxarray = "0.14.1"
+xarray = "2023.4.2"
+distancerasters = "0.3.2"
+opencv-contrib-python = "4.7.0.72"
+dask = "2023.4.0"
+simplejson = "3.19.1"
+Flask-Cors = "3.0.10"
+flask-sock = "0.6.0"
+h11 = "0.14.0"
+websockets = "11.0.2"
+geopandas = "0.12.2"
+pandas = "1.5.3"
```

Comment:
This smells like a `pip freeze` dump!  Are these the top level requirements of your package?

Reply:
As mentioned in my comment for the classification service. Our main goal is just to get all our services connected right now. I'm temporarily using the same dependencies as stated in requirements.txt. Dependencies and packages will be updated once we push towards integrating everything and making sure the websocket routes work.